packages/doc/src/components/GlobalStyle/GlobalStyle.jsx

```diff
@@ -0,0 +1,96 @@
+import { bool } from 'prop-types';
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyle = createGlobalStyle(
+  ({ overflow }) => `
+    #gatsby-focus-wrapper, #___gatsby {
+      height: 100%;
+    }
+
+    html, body {
+      color: #666;
+      font-family: 'neue-haas-grotesk-display';
+      letter-spacing: 0.5px;
+      height: 100%;
+      margin: 0;
+      padding: 0;
+      overflow: ${overflow ? 'hidden' : 'auto'};
```

Comment:
Should we follow the CSS Ordering here? like: 

```
- Layout Properties (position, float, clear, display)
- Box Model Properties (width, height, margin, padding)
- Visual Properties (color, background, border, box-shadow)
- Typography Properties (font-size, font-family, text-align, text-transform)
- Misc Properties (cursor, overflow, z-index)
```

Reply:
Done!